.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Problem
+
+## Expected Behavior
+
+## Actual Behavior
+
+## Steps to Reproduce the Problem
+
+1.
+
+2.
+
+3.
+
+## Specifications
+
+- Version:
+- Platform:
```

Comment:
Platform unnecessary

Reply:
Hmmm, maybe we'll need in this in the future though, I'll get rid of it for now!